src/qibosoq/qick_programs.py

```diff
@@ -34,6 +39,8 @@ def set_bias(self, mode="sweetspot"):
                 i_wf = np.full(duration, value)
                 q_wf = np.zeros(len(i_wf))
                 self.add_pulse(ch, f"const_{value}_{idx}", i_wf, q_wf)
+
+                print(f"Setting bias of ch {ch} to gain {self.max_flux_gain}")
```

Comment:
If possible avoid bare `print`s, and prefer [`logging`](https://docs.python.org/3/library/logging.html).